docs/SNIPPET.txt

```diff
@@ -1,6 +1,9 @@
-#include "hello_world.h"
+#include <stdio.h>
+
+int main(int argc, char* argv[]) {
+
+	for (int i = 1; i <= 5; i++) {
```

Comment:
Can we get rid of the for loop? 
I'd thought the idea of the SNIPPET.md is to provide the simplest program as possible to provide a very basic example. Indeed, I'm assuming this is why you think it should be self-contained? Anyone with no programming experience at all would see a for loop as a pretty arcane piece of symbol garbage.